pkg/build/runner_config.go

```diff
@@ -15,7 +15,11 @@ import (
 
 // TODO: add lock when running interval func
 // 10 second interval
-const DEFAULT_POLL_INTERVAL = "*/10 * * * * *"
+const DEFAULT_BUILD_POLL_INTERVAL = "*/10 * * * * *"
+
+// TODO: add lock when running interval func
+// 1 second interval
+const DEFAULT_JOB_POLL_INTERVAL = "*/1 * * * * *"
```

Comment:
Should we consider moving this somewhere out of build since it is used in running jobs in general, not just builds?